Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -310,7 +359,7 @@ def test_myschedule_put(self , session=None):
             'DESCRIPTION' : put_description,
             'MON_CDE' : 'M',
             'TUE_CDE' : 'T',
-            'WED_CDE' : 'W',
+            'WED_CDE' : None,
```

Comment:
I changed that so that future users of the tests would be able to easily know the value options they have. Yes, it means the meeting is all weekdays except Wednesday.